src/Compilers/Core/Portable/Syntax/SyntaxTriviaList.cs

```diff
@@ -65,7 +67,21 @@ public SyntaxTriviaList(IEnumerable<SyntaxTrivia>? trivias)
         {
         }
 
+        public static SyntaxTriviaList Create(ReadOnlySpan<SyntaxTrivia> trivias)
+        {
+            if (trivias.Length == 0)
+                return default;
+
+            if (trivias.Length == 1)
+                return new SyntaxTriviaList(trivias[0]);
+
+            return new SyntaxTriviaList(token: default, CreateNode(trivias), position: 0, index: 0);
```

Comment:
It looks like these impls are imitating the behavior you would have gotten for 0/1/n arguments to similar SyntaxFactory methods? Is that accurate?

Reply:
correct.